src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -4576,7 +4584,8 @@ void SmartScript::UpdateTimer(SmartScriptHolder& e, uint32 const diff)
             {
                 if (me && me->HasUnitState(UNIT_STATE_CASTING))
                 {
-                    e.timer = 1200;
+                    //e.timer = 1200;
+                    RaisePriority(e);
```

Comment:
the 1200 timer was to emulate Blizz 😭

Reply:
Got it. I didnt delete it because i wasnt sure what it was 😝